review_practice/react-app/src/containers/SearchBar/ModalBody/PriceFilter.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useContext } from 'react';
+
+import { SearchRoomContext } from 'pages/SearchRoom';
+
+import PriceRangeSlider from 'components/PriceRangeSlider';
+
+const MIN_PRICE = 0;
+const MAX_PRICE = 100000;
```

Comment:
```suggestion
const MAX_PRICE = 100_000;
```
이제 JS에서도 이런 표현을 사용할 수 있답니다!